src/components/Pagination/Pagination.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import Button from '../Button/Button';
+import styles from './Pagination.module.css';
+
+export default class Pagination extends Component {
+
+  render() {
+    const {
+      getNextPage,
+      getPrevPage,
+      updateItemsPerPage,
+      parentState,
+      page,
+      pages
+    } = this.props;
+    return (
+      <div className={styles.main}>
+        <Button onClick={() => getPrevPage(parentState)}>Previous</Button>
+        <select onChange={(e) => updateItemsPerPage(e.target.value)} name="itemsPerPage" id="itemsPerPage">
```

Comment:
on the future, it will be better to create component Select, and add props options to this component and render tag option in options.map